### PR TITLE
Fix video volume issue and progress bar stats

### PR DESCRIPTION
**Pull request by SYBIOTE**
This pull request fixes the issue with the video volume and updates the progress bar to show accurate statistics.

### CODE REVIEW

File: packages/ui/src/components/editor/properties/video/index.tsx [hunk 1]
```diff
@@ -99,7 +105,11 @@ export const VideoNodeEditor: EditorComponentType = (props) => {
       description={t('editor:properties.video.description')}
       icon={<VideoNodeEditor.iconComponent />}
     >
-      <ProgressBar value={5} paused={false} totalTime={100} />
+      <ProgressBar
```
Comment (HexaField) [inline packages/ui/src/components/editor/properties/video/index.tsx]: progress bare doesnt update since this data is not reactive

**Review by HexaField — CHANGES_REQUESTED** [review]

File: packages/ui/src/components/editor/properties/video/index.tsx (hunk 1 above)
Comment (SYBIOTE) [reply packages/ui/src/components/editor/properties/video/index.tsx]: Yes, that's true, I still left it in as we can obtain the duration of the Clip from this. Ill remove it nonetheless

File: packages/ui/src/components/editor/input/Progress/index.tsx [hunk 2]
```diff
@@ -34,10 +34,11 @@ export interface ProgressBarProps extends ProgressProps {
 }
 
 export default function ProgressBar({ value, paused, totalTime, ...rest }: ProgressBarProps) {
+  console.log('DEBUG value', value)
```
Comment (HexaField) [inline packages/ui/src/components/editor/input/Progress/index.tsx]: remove console log

**Review by HexaField — DISMISSED** [review]

File: packages/ui/src/components/editor/input/Progress/index.tsx (hunk 2 above)
Comment (SYBIOTE) [reply packages/ui/src/components/editor/input/Progress/index.tsx]: Done

**Comment by AidanCaruso** [comment]
@SYBIOTE I'll give an approving review because this is out of scope, but we really need to fix the audio seemingly starting at 1 in a range of 1-100, where 1 is full volume and 100 is subwoofer-breaking. Shown in the video - warning, it's really loud.

https://github.com/user-attachments/assets/019cf33a-cff9-4356-b7dc-8fc7a0043698

**Review by AidanCaruso — APPROVED** [review]

**Comment by SYBIOTE** [comment]
oh, cool ill fix it now

**Pull request merged**